release/release_gold.sh

```diff
@@ -20,23 +34,29 @@ function copy_rc {
 	ssh -i lrdcom-vm-1 root@lrdcom-vm-1 cp -a "/www/releases.liferay.com/dxp/release-candidates/${LIFERAY_RELEASE_VERSION}-${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}" "/www/releases.liferay.com/dxp/${LIFERAY_RELEASE_VERSION}"
 }
 
+function get_release_properties_value {
+    grep "${1}" ${file} | cut -d'=' -f2
```

Comment:
AWK could do it without a pipe. However, I am not sure if it was more readable or usable.

Reply:
Do we not have a function for this purpose already?